File: 2485 find-the-pivot-integer/solution.cs
public class Solution {
    public int PivotInteger(int n) {
        int sum=n*(n+1)/2;
        int lsum=0;
        for(int i=1;i<=n;i++) {
            if(lsum==sum-lsum-i) return i;
            lsum+=i;
        }
        return -1;
    }
}

File: 2485 find-the-pivot-integer/solution.go
func pivotInteger(n int) int {
    sum:=n*(n+1)/2
    lsum:=0
    for i:=1;i<=n;i++ {
        if lsum==sum-i-lsum {
            return i
        }
        lsum+=i
    }
    return -1
}
